lib/SIL/IR/SILBasicBlock.cpp

```diff
@@ -26,12 +26,24 @@
 #include "swift/SIL/SILModule.h"
 #include "swift/SIL/SILUndef.h"
 #include "swift/Strings.h"
+extern "C" {
+#include "swift/SIL/SILBridging.h"
```

Comment:
Can we just put the extern "C" in swift/SIL/SILBridging.h and `#ifdef __cplusplus`? The includers of a header in c++ shouldn't need to know to do this. They should just be able to include SILBridging.h.